platforms/minigame/platforms/bytedance/wrapper/engine/VideoPlayer.js

```diff
@@ -41,6 +41,9 @@ if (cc.internal.VideoPlayer) {
     class VideoPlayerImplMiniGame extends cc.internal.VideoPlayerImpl {
         constructor (componenet) {
             super(componenet);
+            this.cameraNode = new cc.Node();
```

Comment:
Why call it `cameraNode`? I didn't see any camera stuff, it's just a Node that contains a Sprite component.

Reply:
This is written with reference to the official website documentation. I feel similar to camera playing video

Reply:
But it's not a camera, I think it's better to rename it to `videoNode` or `videoTextureNode` or `textureNode`.

Reply:
ok